extensions/amp-accordion/1.0/accordion.js

```diff
@@ -136,20 +151,28 @@ export function AccordionSection({
   headerAs: HeaderComp = 'header',
   contentAs: ContentComp = 'div',
   expanded: defaultExpanded = false,
+  animate: defaultAnimate = false,
   header,
   children,
   ...rest
 }) {
   const [id] = useState(generateSectionId);
   const [expandedState, setExpandedState] = useState(defaultExpanded);
+  const contentRef = useRef(null);
+  const hasMountedRef = useRef(false);
 
-  const {registerSection, isExpanded, toggleExpanded} = useContext(
+  const {registerSection, isAnimated, isExpanded, toggleExpanded} = useContext(
     AccordionContext
   );
 
+  useEffect(() => {
+    hasMountedRef.current = true;
+    return () => (hasMountedRef.current = false);
+  }, []);
+
   useLayoutEffect(() => {
     if (registerSection) {
-      registerSection(id, defaultExpanded);
+      return registerSection(id, defaultExpanded);
```

Comment:
Is this supposed to be cleanup now?

Reply:
Yes, register also does cleanup. I added a test for this as well. It was simply missed from the previous PR.